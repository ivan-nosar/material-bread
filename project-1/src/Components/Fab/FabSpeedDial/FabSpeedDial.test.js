import React from 'react';
import { FabSpeedDial } from '../../..';
import ShallowRenderer from 'react-test-renderer/shallow';
const renderer = new ShallowRenderer();
const actions = [
  {
    text: 'Favorites',
  },
  {
    text: 'Language',
  },
  {
    text: 'Location',
  },
  {
    text: 'Video',
  },
];

test('FabSpeedDial Renders', () => {
  const tree = renderer.render(<FabSpeedDial actions={actions} />);
  expect(tree).toMatchSnapshot();
});
