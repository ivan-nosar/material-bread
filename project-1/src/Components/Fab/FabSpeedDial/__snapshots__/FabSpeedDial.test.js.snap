// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FabSpeedDial Renders 1`] = `
<Context.Consumer>
  [Function]
</Context.Consumer>
`;
